### ISSUE TITLE
Wont work on Google Chrome on windows.

### ISSUE THREAD

**Issue by rushabrdx** [razorpay/razorpay-prestashop#43]
Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
https://imgur.com/a/RrvBKPn
Works fine on mobile though.

**Comment by ramth05** [comment]
> Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> https://imgur.com/a/RrvBKPn
> Works fine on mobile though.

@rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order

**Comment by rushabrdx** [comment]
> > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > https://imgur.com/a/RrvBKPn
> > Works fine on mobile though.
> 
> @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order

Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .

**Comment by rushabrdx** [comment]
Side not works perfectly fine on my android phone tough

**Comment by ramth05** [comment]
> > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > https://imgur.com/a/RrvBKPn
> > > Works fine on mobile though.
> > 
> > 
> > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> 
> Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .

@rushabrdx , looks like plugin not updated properly.

**Comment by rushabrdx** [comment]
> > > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > > https://imgur.com/a/RrvBKPn
> > > > Works fine on mobile though.
> > > 
> > > 
> > > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> > 
> > 
> > Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .
> 
> @rushabrdx , looks like plugin not updated properly.

Deleted and reinstalled from scratch. still wont work (on windows)

**Comment by ramth05** [comment]
> > > > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > > > https://imgur.com/a/RrvBKPn
> > > > > Works fine on mobile though.
> > > > 
> > > > 
> > > > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> > > 
> > > 
> > > Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .
> > 
> > 
> > @rushabrdx , looks like plugin not updated properly.
> 
> Deleted and reinstalled from scratch. still wont work (on windows)

@rushabrdx, Can you please check error logs in your server. If still not fine can you please create a test/clone site in your server and pass the information to us to look into it further.

**Comment by rushabrdx** [comment]
> > > > > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > > > > https://imgur.com/a/RrvBKPn
> > > > > > Works fine on mobile though.
> > > > > 
> > > > > 
> > > > > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> > > > 
> > > > 
> > > > Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .
> > > 
> > > 
> > > @rushabrdx , looks like plugin not updated properly.
> > 
> > 
> > Deleted and reinstalled from scratch. still wont work (on windows)
> 
> @rushabrdx, Can you please check error logs in your server. If still not fine can you please create a test/clone site in your server and pass the information to us to look into it further.

Only found this 2020/01/28 - 13:26:28: Can't load Order status at line 242 in file classes/PaymentModule.php

**Comment by ramth05** [comment]
> > > > > > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > > > > > https://imgur.com/a/RrvBKPn
> > > > > > > Works fine on mobile though.
> > > > > > 
> > > > > > 
> > > > > > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> > > > > 
> > > > > 
> > > > > Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .
> > > > 
> > > > 
> > > > @rushabrdx , looks like plugin not updated properly.
> > > 
> > > 
> > > Deleted and reinstalled from scratch. still wont work (on windows)
> > 
> > 
> > @rushabrdx, Can you please check error logs in your server. If still not fine can you please create a test/clone site in your server and pass the information to us to look into it further.
> 
> Only found this 2020/01/28 - 13:26:28: Can't load Order status at line 242 in file classes/PaymentModule.php

@rushabrdx , Can you please create clone  test site and pass the FTP and backend details to us. you can email me the details.

**Comment by rushabrdx** [comment]
> > > > > > > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > > > > > > https://imgur.com/a/RrvBKPn
> > > > > > > > Works fine on mobile though.
> > > > > > > 
> > > > > > > 
> > > > > > > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> > > > > > 
> > > > > > 
> > > > > > Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .
> > > > > 
> > > > > 
> > > > > @rushabrdx , looks like plugin not updated properly.
> > > > 
> > > > 
> > > > Deleted and reinstalled from scratch. still wont work (on windows)
> > > 
> > > 
> > > @rushabrdx, Can you please check error logs in your server. If still not fine can you please create a test/clone site in your server and pass the information to us to look into it further.
> > 
> > 
> > Only found this 2020/01/28 - 13:26:28: Can't load Order status at line 242 in file classes/PaymentModule.php
> 
> @rushabrdx , Can you please create clone test site and pass the FTP and backend details to us. you can email me the details.

I have no idea how to clone a test site for you? i can set up prestashop from scratch if you like .

**Comment by ramth05** [comment]
> > > > > > > > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > > > > > > > https://imgur.com/a/RrvBKPn
> > > > > > > > > Works fine on mobile though.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> > > > > > > 
> > > > > > > 
> > > > > > > Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .
> > > > > > 
> > > > > > 
> > > > > > @rushabrdx , looks like plugin not updated properly.
> > > > > 
> > > > > 
> > > > > Deleted and reinstalled from scratch. still wont work (on windows)
> > > > 
> > > > 
> > > > @rushabrdx, Can you please check error logs in your server. If still not fine can you please create a test/clone site in your server and pass the information to us to look into it further.
> > > 
> > > 
> > > Only found this 2020/01/28 - 13:26:28: Can't load Order status at line 242 in file classes/PaymentModule.php
> > 
> > 
> > @rushabrdx , Can you please create clone test site and pass the FTP and backend details to us. you can email me the details.
> 
> I have no idea how to clone a test site for you? i can set up prestashop from scratch if you like .

@rushabrdx , we are fine with new setup in your server as well. Please use older version of plugin for your live site for time being( as new plugin only fixed issue related with promo-code , which only some merchants facing not all) .

**Comment by rushabrdx** [comment]
> > > > > > > > > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > > > > > > > > https://imgur.com/a/RrvBKPn
> > > > > > > > > > Works fine on mobile though.
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .
> > > > > > > 
> > > > > > > 
> > > > > > > @rushabrdx , looks like plugin not updated properly.
> > > > > > 
> > > > > > 
> > > > > > Deleted and reinstalled from scratch. still wont work (on windows)
> > > > > 
> > > > > 
> > > > > @rushabrdx, Can you please check error logs in your server. If still not fine can you please create a test/clone site in your server and pass the information to us to look into it further.
> > > > 
> > > > 
> > > > Only found this 2020/01/28 - 13:26:28: Can't load Order status at line 242 in file classes/PaymentModule.php
> > > 
> > > 
> > > @rushabrdx , Can you please create clone test site and pass the FTP and backend details to us. you can email me the details.
> > 
> > 
> > I have no idea how to clone a test site for you? i can set up prestashop from scratch if you like .
> 
> @rushabrdx , we are fine with new setup in your server as well. Please use older version of plugin for your live site for time being( as new plugin only fixed issue related with promo-code , which only some merchants facing not all) .

Done the clone site. please give me your mail id to forward details

**Comment by ramth05** [comment]
> > > > > > > > > > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > > > > > > > > > https://imgur.com/a/RrvBKPn
> > > > > > > > > > > Works fine on mobile though.
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .
> > > > > > > > 
> > > > > > > > 
> > > > > > > > @rushabrdx , looks like plugin not updated properly.
> > > > > > > 
> > > > > > > 
> > > > > > > Deleted and reinstalled from scratch. still wont work (on windows)
> > > > > > 
> > > > > > 
> > > > > > @rushabrdx, Can you please check error logs in your server. If still not fine can you please create a test/clone site in your server and pass the information to us to look into it further.
> > > > > 
> > > > > 
> > > > > Only found this 2020/01/28 - 13:26:28: Can't load Order status at line 242 in file classes/PaymentModule.php
> > > > 
> > > > 
> > > > @rushabrdx , Can you please create clone test site and pass the FTP and backend details to us. you can email me the details.
> > > 
> > > 
> > > I have no idea how to clone a test site for you? i can set up prestashop from scratch if you like .
> > 
> > 
> > @rushabrdx , we are fine with new setup in your server as well. Please use older version of plugin for your live site for time being( as new plugin only fixed issue related with promo-code , which only some merchants facing not all) .
> 
> Done the clone site. please give me your mail id to forward details

@rushabrdx , ram.techchef@razorpay.com

**Comment by rushabrdx** [comment]
> > > > > > > > > > > > Getting a new error with this module. Nothing would happen when you click pay no matter how many times. Heres an screenshot of error from google console.
> > > > > > > > > > > > https://imgur.com/a/RrvBKPn
> > > > > > > > > > > > Works fine on mobile though.
> > > > > > > > > > > 
> > > > > > > > > > > 
> > > > > > > > > > > @rushabrdx , can you please check looks like this url getting blocked by your server https://siddhisilver.in/module/razorpay/order
> > > > > > > > > > 
> > > > > > > > > > 
> > > > > > > > > > Yes this wont open and will throw 500 error, altough i havent done anything. it was running fine until yesterday, today updated it to latest version and now it wont work .
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > @rushabrdx , looks like plugin not updated properly.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Deleted and reinstalled from scratch. still wont work (on windows)
> > > > > > > 
> > > > > > > 
> > > > > > > @rushabrdx, Can you please check error logs in your server. If still not fine can you please create a test/clone site in your server and pass the information to us to look into it further.
> > > > > > 
> > > > > > 
> > > > > > Only found this 2020/01/28 - 13:26:28: Can't load Order status at line 242 in file classes/PaymentModule.php
> > > > > 
> > > > > 
> > > > > @rushabrdx , Can you please create clone test site and pass the FTP and backend details to us. you can email me the details.
> > > > 
> > > > 
> > > > I have no idea how to clone a test site for you? i can set up prestashop from scratch if you like .
> > > 
> > > 
> > > @rushabrdx , we are fine with new setup in your server as well. Please use older version of plugin for your live site for time being( as new plugin only fixed issue related with promo-code , which only some merchants facing not all) .
> > 
> > 
> > Done the clone site. please give me your mail id to forward details
> 
> @rushabrdx , [ram.techchef@razorpay.com](mailto:ram.techchef@razorpay.com)

Done . Please put shop back in maintaince mode once your done otherwise google search results will show them too thanks.

**Comment by shivakadakol** [comment]
i have some problem. whenever click on razorpay nothing display page.

**Comment by ramth05** [comment]
@shivakadakol , please open new issue.
